namespaces/kubernetes.yml

```diff
@@ -94,6 +102,117 @@ namespace:
         description: >
           The provider associated with this Kubernetes object
 
+  - name: event
+    type: group
+    description: >
+      The kubernetes event obtained from kubernetes master API
+      The event is already JSON object and as whole nested under kubernetes field
+      This description should loosely follow 'type Event' in https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go
+    fields:
+      - name: verb
+        type: string
+        example: ADDED
+        description: >
+          The type of event, can be ADDED, MODIFIED, DELETED
+      - name: metadata
+        type: group
+        description: >
+          Information related to the location and time of the event creation
+        fields:
+          - name: name
+            type: string
+            example: java-mainclass-1.14d888a4cfc24890
+            description: >
+              Name of the object that triggered the event creation
+          - name: namespace
+            type: string
+            example: default
+            description: >
+              The name of the namespace which induced the event
+              It differs from namespace_name, which will be in case of every event the 'eventrouter'
+          - name: selfLink
+            type: string
+            example: /api/v1/namespaces/javaj/events/java-mainclass-1.14d888a4cfc24890
+            description: >
+              Link to the event itself
+          - name: uid
+            type: string
+            example: d828ac69-7b58-11e7-9cf5-5254002f560c
+            description: >
+              Event's unique ID
+          - name: resourceVersion
+            type: integer
+            example: 311987
+            description: >
+              String that identifies the server's internal version of the event that can be used by clients to determine when objects have changed
+          - name: creationTimestamp
+            type: date
+            format: yyyy-MM-dd HH:mm:ss,SSSZ||yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ||yyyy-MM-dd'T'HH:mm:ssZ||dateOptionalTime
+            example: 2017-08-07T10:11:57Z
+            description: >
+              UTC value marking when the event was created
+      - name: involvedObject
+        type: group
+        description: >
+          Description of the object involved in the event creation
+        fields:
+          - name: kind
+            type: string
+            example: ReplicationController
+            description: >
+              Type of the object
+          - name: namespace
+            type: string
+            example: default
+            description: >
+              The name of the namespace in which the object triggered the event
+              In case this event is not triggered by a pod then it differs from kubernetes.namespace_name, which will be in case of every event eventrouter's namespace
```

Comment:
Perhaps consider the wording the above two lines as: "The namespace of the object that triggered the event.  If the event is not triggered by a pod, this namespace value will have a different value than `kubernetes.namespace_name`."